Letterbook.Web.Tests/Letterbook.Web.Tests.csproj

```diff
@@ -33,4 +34,10 @@
       <ProjectReference Include="..\Letterbook.Web\Letterbook.Web.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Reference Include="ServiceStack">
```

Comment:
do we actually need this for something? ServiceStack powers the SSG for the docs site, but I kind of hate it and I want to get rid of it eventually, not pull it into the main app.

Reply:
Ahm, good question, I'll check

Reply:
Okay looks like were okay without that reference